2020-01-22 02:48:53 INFO  Operation directory: /Users/johnwalker/projects/baseball/WBBaseballStats/data/Scripts/logs/LOAD_20200122-024853-791499
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.43.130:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.43.130:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.43.130:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.33.199:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.33.199:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.33.199:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.33.199:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.33.199:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.33.199:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.33.199:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.33.199:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.33.199:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.43.130:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.43.130:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.43.130:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.43.130:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.43.130:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.43.130:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.43.130:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.43.130:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.43.130:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.33.199:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.33.199:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.33.199:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.43.130:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.43.130:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.43.130:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.33.199:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.33.199:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.33.199:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.43.130:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.43.130:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.43.130:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.33.199:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.33.199:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.33.199:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.33.199:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.33.199:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.33.199:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.33.199:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.33.199:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.33.199:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.43.130:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.43.130:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.43.130:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating connection to /172.31.43.130:9042
com.datastax.driver.core.exceptions.TransportException: [/172.31.43.130:9042] Cannot connect
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:215)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
Caused by: io.netty.channel.ConnectTimeoutException: connection timed out: /172.31.43.130:9042
	at java.lang.Thread.run(Thread.java:748) [8 skipped]
2020-01-22 02:49:00 WARN  Error creating pool to /172.31.33.199:9042
com.datastax.driver.core.exceptions.ConnectionException: [/172.31.33.199:9042] Pool was closed during initialization
	at com.datastax.driver.core.HostConnectionPool$2.onSuccess(HostConnectionPool.java:164)
	at com.datastax.driver.core.HostConnectionPool$2.onSuccess(HostConnectionPool.java:149)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:213) [34 skipped]
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
2020-01-22 02:49:00 WARN  Error creating pool to /172.31.43.130:9042
com.datastax.driver.core.exceptions.ConnectionException: [/172.31.43.130:9042] Pool was closed during initialization
	at com.datastax.driver.core.HostConnectionPool$2.onSuccess(HostConnectionPool.java:164)
	at com.datastax.driver.core.HostConnectionPool$2.onSuccess(HostConnectionPool.java:149)
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:213) [34 skipped]
	at com.datastax.driver.core.Connection$1.operationComplete(Connection.java:185)
	at java.lang.Thread.run(Thread.java:748) [15 skipped]
2020-01-22 02:49:40 INFO  Operation LOAD_20200122-024853-791499 completed successfully in 39 seconds.
2020-01-22 02:49:42 INFO  Final stats:
2020-01-22 02:49:42 INFO  Records: total: 104,324, successful: 104,324, failed: 0
2020-01-22 02:49:42 INFO  Batches: total: 96,631, size: 1.10 mean, 1 min, 4 max
2020-01-22 02:49:42 INFO  Memory usage: used: 77 MB, free: 211 MB, allocated: 288 MB, available: 3,641 MB, total gc count: 17, total gc time: 131 ms
2020-01-22 02:49:42 INFO  Writes: total: 104,324, successful: 104,324, failed: 0, in-flight: 0
2020-01-22 02:49:42 INFO  Throughput: 2,499 writes/second
2020-01-22 02:49:42 INFO  Latencies: mean 185.60, 75p 235.93, 99p 905.97, 999p 1,568.67 milliseconds
2020-01-22 02:49:42 INFO  Last processed positions can be found in positions.txt
